Add subsection for custom CA in air-gapped environments (#2030)
elastic/observability-docs: docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc

For this to work, the {kib} server must connect to `https://epr.elastic.co` on port `443`.

[discrete]
[[fleet-errors-tls]]
== {kib} cannot connect to {package-registry} in air-gapped environments

In air-gapped environments where custom Certificater Authorities (CA) are used, if the custom

> CA is not available for {kib}, you can see error like this one:

Reviewer: I'd change the word order a bit to shorten the sentence. BTW, I'm not a big fan of "encounter" instead of "see" but we've been asked to avoid using ableist language.

> In air-gapped environments where custom Certificater Authorities (CA) are used, if the custom
> CA is not available for {kib}, you can see error like this one:

Reviewer: ```suggestion
In air-gapped environments, you may encounter the following error if you're using a custom Certificate Authority (CA) that is not available to {kib}:
```

Reviewer: I'd change the word order a bit to shorten the sentence. BTW, I'm not a big fan of "encounter" instead of "see" but we've been asked to avoid using ableist language.

[source,json]
---
{"type":"log","@timestamp":"2022-03-02T09:58:36-05:00","tags":["error","plugins","fleet"],"pid":58716,"message":"Error connecting to package registry: request to https://customer.server.name:8443/categories?experimental=true&include_policy_templates=true&kibana.version=7.17.0 failed, reason: self signed certificate in certificate chain"}
---

This can be addressed by adding your CA certificate file path to the {kib} startup

> file by defining the `NODE_EXTRA_CA_CERTS` environment variable.

Reviewer 2: Good idea to put this here :+1:

> This can be addressed by adding your CA certificate file path to the {kib} startup
> file by defining the `NODE_EXTRA_CA_CERTS` environment variable.

Reviewer: ```suggestion
To fix this problem, add your CA certificate file path to the {kib} startup
file by defining the `NODE_EXTRA_CA_CERTS` environment variable.
```

> 

Author: How can I set a link to the other page ? So the user could see more information about this.

Reviewer 2: Try with `<<air-gapped>>`, or `<<air-gapped-tls>>` for the section about TLS.

Author: Perfect, I'll add a link with that notation, thanks!
